Tests/configure_and_test_integration_instances.py

```diff
@@ -757,7 +758,14 @@ def set_marketplace_gcp_bucket_for_build(client, prints_manager, branch_name, ci
         'marketplace.initial.sync.delay': '0',
         'content.pack.ignore.missing.warnings.contentpack': 'true'
     }
-    if not is_nightly:
+    if is_private:
+        server_configuration['marketplace.bootstrap.bypass.url'] = 'https://storage.googleapis.com/marketplace-ci-build'
```

Comment:
i think we should combine the actions here with the else, most of them are actually identical, apart from the GW